.github/workflows/dart_vm.yaml

```diff
@@ -39,11 +39,26 @@ jobs:
       - name: Git Submodules
         run: git submodule update --init
 
+      # Some Dart packages require Flutter to be available, for example `amplify_core`
+      # requires it for its `doc` package. Running this before "Setup Dart" means that
+      # the `dart` executable in PATH will always be the one setup by "Setup Dart".
+      #
+      # TODO(dnys1): Create mapping of Flutter->Dart versions so we can just use Dart
+      # vended by Flutter.
+      - name: Setup Flutter
```

Comment:
Yeah. Even though `amplify_core` is Dart-only, the examples we showcase in its docs all reference Flutter packages. So the `doc` package in `amplify_core` needs a dependency on Flutter so that we can have Flutter-specific samples in the `amplify_core` docs.